modules/Publish/module.targets

```diff
@@ -0,0 +1,36 @@
+<Project>
+  <PropertyGroup>
+    <PublishDependsOn>$(PublishDependsOn);PublishToAzure</PublishDependsOn>
+
+    <PushToBlobFeed_Overwrite Condition="'$(PushToBlobFeed_Overwrite)' == ''">false</PushToBlobFeed_Overwrite>
+    <PushToBlobFeed_MaxClients Condition="'$(PushToBlobFeed_MaxClients)' == ''">8</PushToBlobFeed_MaxClients>
+    <FileRelativePathBase Condition="'$(FileRelativePathBase)' == ''">assets</FileRelativePathBase>
+    <FileRelativePathBase Condition="!HasTrailingSlash('$(FileRelativePathBase)')">$(FileRelativePathBase)/</FileRelativePathBase>
+  </PropertyGroup>
+
+  <Target Name="Publish" DependsOnTargets="$(PublishDependsOn)" />
+
+  <Target Name="PublishToAzure" DependsOnTargets="GetArtifactInfo" Condition="'$(PublishBlobFeedUrl)' != ''">
+    <ItemGroup>
+      <PackagesToPublish Include="%(ArtifactInfo.Identity)" Condition="'%(ArtifactInfo.ArtifactType)' == 'NuGetPackage'" />
+      <FilesToPublish Include="%(ArtifactInfo.Identity)" Condition="'%(ArtifactInfo.AzureAssetsPath)' != ''">
+        <RelativeBlobPath>$(FileRelativePathBase)%(ArtifactInfo.AzureAssetsPath)</RelativeBlobPath>
+      </FilesToPublish>
+    </ItemGroup>
+
+    <PushToBlobFeed ExpectedFeedUrl="$(PublishBlobFeedUrl)"
```

Comment:
Could we get a little comment in here saying what these two Push tasks are trying to accomplish? It's not super clear to me just from the parameters used.

Reply:
Yup, I added a comments pointing to https://github.com/dotnet/buildtools/tree/master/src/Microsoft.DotNet.Build.Tasks.Feed for more details.